app.py

```diff
@@ -0,0 +1,92 @@
+import os
+import sys
+import spacy
+import re
+from polyglot.text import Text
+from polyglot.detect import Detector
+from flask import Flask, request, jsonify, abort
+nlp = spacy.load('xx_ent_wiki_sm')
```

Comment:
flake8 can't possibly agree to this lack of whitespace. Are you running it right?

Reply:
Flake8 is happy with it. I also don't know where you're missing whitespace with these lines.